src/common/EmptyState.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
What is this for?

Reply:
A generic component for empty states - e.g when the patient has no files/ the doctor has no patients etc